syz-manager/manager.go

```diff
@@ -1086,7 +1086,7 @@ func (mgr *Manager) saveCrash(crash *Crash) bool {
 const maxReproAttempts = 3
 
 func (mgr *Manager) needLocalRepro(crash *Crash) bool {
-	if !mgr.cfg.Reproduce || crash.Corrupted || crash.Suppressed {
```

Comment:
Why not let `mgr.cfg.Reproduce` stay here?

Reply:
Restored. Also added additional check to saveCrash.